components/Header/Hamburger.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface HamburgerProps {
+  isOpen: boolean;
+  toogle: () => void;
+}
+
+const Hamburger: React.FC<HamburgerProps> = ({ isOpen, toogle }) => {
+  return (
```

Comment:
brzydko ta interpolacja w stringu wygląda, zróbmy to za pomocą paczkie `clsx`,
https://www.npmjs.com/package/clsx
```jsx
const buttonStyles = clsx('hamburger-bar', isOpen && 'transform rotate-45 -translate-x-00 translate-y-2');

<button className={buttonStyles} ...
```

Reply:
zrobione, tylko nie wiem czy dobrze nazwy klas dobrałem (potrzeba 4 różne stałe zdefiniować).